packages/block-editor/src/components/block-list/content.scss

```diff
@@ -185,7 +185,7 @@ _::-webkit-full-page-media, _:future, :root .has-multi-selection .block-editor-b
 
 	&.has-negative-margin {
 		&.is-selected,
-		&.has-child-selected {
+		&:has(.is-selected) {
```

Comment:
@MaggieCabrera Would appreciate if you could test if this still works correctly 🙂

Reply:
I think we need both selectors, did you remove the right line? shouldn't it remove line 187 instead of 188?

Reply:
Yes, it's correct: `.has-child-selected` is replaced with `:has(.is-selected)`